### PR TITLE
fix -timedemo

### PR DESCRIPTION
**Pull request by rfomin**

### CODE REVIEW

File: Source/d_main.c [hunk 1]
```diff
@@ -2356,7 +2360,7 @@ void D_DoomMain(void)
         {
           I_StartTic ();
           D_ProcessEvents ();
-          G_BuildTiccmd (&netcmds[consoleplayer]);
+          G_BuildTiccmd (netcmds);
```
Comment (fabiangreffrath) [inline Source/d_main.c:2363]: I don't get why timedemo was broken before this change? Could you explain?

Comment (rfomin) [reply Source/d_main.c:2363]: In MBF `netcmds` is initialized as a static global array, in Choco it's just a pointer. Choco has `ticdata` which is similar to MBF's `netcmds`:
https://github.com/fabiangreffrath/woof/blob/1f2c476af169cf07fea04149f2cb19ad42d92fd7/Source/d_loop.c#L64

Comment (rfomin) [reply Source/d_main.c:2363]: Also `G_BuildTiccmd(ticcmd_t* cmd)` initialize `cmd` parameter with this code:
https://github.com/fabiangreffrath/woof/blob/1f2c476af169cf07fea04149f2cb19ad42d92fd7/Source/g_game.c#L288-L289
"Historical" code I guess :smile:

Comment (fabiangreffrath) [reply Source/d_main.c:2363]: Thank you!

Comment (rfomin) [reply Source/d_main.c:2363]: Why do we need `G_BuildTiccmd` in `singletics` mode? We have no game input. Is this for a more honest FPS measurement?

**Pull request merged**